xls/modules/zstd/frame_header_test.cc

```diff
@@ -57,10 +57,21 @@ enum FrameHeaderStatus : uint8_t {
   UNSUPPORTED_WINDOW_SIZE
 };
 
-struct ZstdFrameHeader {
-  absl::Span<const uint8_t> kBuffer;
+class ZstdFrameHeader {
+ public:
   ZSTD_frameHeader kHeader;
   size_t kResult;
```

Comment:
These members should be private with accessors and shouldn't be named kSomething because they're not constexpr.

Reply:
Fixed